app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDao.java

```diff
@@ -24,8 +24,9 @@
     @Query("SELECT * FROM event ORDER BY start")
     List<Event> getAll();
 
-    @Query("SELECT id FROM event ORDER BY id DESC LIMIT 1")
-    int getLatestId();
+    // Get earliest event
+    @Query("SELECT * FROM event ORDER BY start LIMIT 1")
```

Comment:
You want all events *after* today. So you will need to add a `where` clause. Then the function name also fits better ;)